contracts/stellar-example/src/tests/test.rs

```diff
@@ -180,78 +180,199 @@ fn gmp_example() {
 fn its_example() {
     let env = Env::default();
 
-    let user = Address::generate(&env).to_string_bytes();
+    let user = Address::generate(&env);
 
+    let original_source_chain = String::from_str(&env, SOURCE_CHAIN_NAME);
     let TestConfig {
-        signers,
-        gateway_client,
-        its_client,
-        app: example_app,
+        its_client: original_source_its_client,
+        app: original_source_app,
```

Comment:
```suggestion
        its_client: source_its_client,
        app: source_app,
```
Why add original prefix? There's no third Hub chain in this test